tests/test_base_plugin.py

```
@@ -1225,33 +1257,35 @@ def test_strip_raw_comprehensive(self):
         result = plugin.strip_raw(nested_data)
         self.assertEqual(result, {"data": {"other": "value"}, "normal": "data"})
 
-    @patch("mmrelay.meshtastic_utils.connect_meshtastic")
     @patch("mmrelay.plugins.base_plugin.queue_message")
-    def test_send_message_no_client(self, mock_queue, mock_connect):
-        """Test send_message when no meshtastic client available (lines 431-432)."""
-        mock_connect.return_value = None
+    @patch("mmrelay.plugins.base_plugin.get_radio_registry")
+    def test_send_message_no_client(self, mock_get_registry, mock_queue):
```

Ruff (0.14.14)

Unused method argument: `mock_queue`

(ARG002)